src/test/java/io/odh/test/e2e/upgrade/OlmUpgradeST.java

```diff
@@ -113,8 +138,38 @@ void testUpgradeOlm() throws IOException, InterruptedException {
         Instant operatorLogCheckTimestamp = Instant.now();
 
         // Verify that NTB pods are stable
-        PodUtils.waitForPodsReady(ntbNamespace, lblSelector, 1, true, () -> { });
+        PodUtils.waitForPodsReady(ntbNamespace, lblSelectorPre, 1, true, () -> { });
         // Check logs in operator pod
         UpgradeUtils.deploymentLogIsErrorEmpty(olmInstall.getNamespace(), olmInstall.getDeploymentName(), operatorLogCheckTimestamp);
+
+        // RHOAIENG-10827: creating new notebooks after upgrade restarted existing notebooks
+        Allure.step("Deploy another notebook after upgrade");
+        deployNotebook(ntbNamespace, ntbNamePost);
```

Comment:
Nitpick: you're using `Post` and `After`, maybe we should choose just one word in scope of this test.